.github/ISSUE_TEMPLATE/chore-task.md

```
@@ -1,32 +1,28 @@
 ---
 name: "🚀 Chore"
-about: 리팩토링/빌드/의존성/CI/CD 등 비기능적 작업
+about: Non-functional tasks such as refactoring/build/dependency/CI/CD
 title: "🚀 [Chore] "
 labels: chore
 assignees: ''
 ---
 
-## 🚀 작업 내용
-무엇을 정리/개선/변경하려는지 적어주세요.
+## 🚀 Task Description
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)